models/object_detection_nanodet/demo.py

```diff
@@ -0,0 +1,110 @@
+import cv2
+import numpy as np
+import argparse
+import time
+import matplotlib.pyplot as plt
```

Comment:
Use `cv2.imshow` to display result image. Take the following as an exmaple:
https://github.com/opencv/opencv_zoo/blob/54190541bb9f717144b255358ca6b37e3e47bf23/models/face_detection_yunet/demo.py#L106-L110

Reply:
reviewed changes, 
- replaced plt.imshow() with cv2.imshow()